extensions/wikia/UserLogin/js/UserLoginAjaxForm.js

```diff
@@ -87,8 +96,15 @@
 	 * @param {string} html
 	 */
 	UserLoginAjaxForm.prototype.retrieveTemplateCallback = function (html) {
-		var content = $('<div>').hide().append(html),
-			form = this.form;
+		var content, form;
+
+		if (typeof this.options.retrieveTemplateCallback === 'function') {
```

Comment:
Would you suggest just checking to see if `typeof this.options.retrieveTemplateCallback !== undefined`? Either way we have to check if it's been passed, since it's an optional setting, and in most scenarios it will be undefined. 

I agree that it's not super great code, but i've actually been refactoring and deleting these optional callback functions as much as possible. This one is still around tho for the change password flow inside a modal. Actually, it might be better to trigger an event instead...